licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/decision/ConsiderationSpec.groovy

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.digital.hmpps.licences.specs.decision
+
+import geb.spock.GebReportingSpec
+import spock.lang.Shared
+import spock.lang.Stepwise
+import uk.gov.justice.digital.hmpps.licences.pages.TaskListPage
+import uk.gov.justice.digital.hmpps.licences.pages.decision.ApprovalConsiderationPage
+import uk.gov.justice.digital.hmpps.licences.pages.decision.ApprovalReleasePage
+import uk.gov.justice.digital.hmpps.licences.util.Actions
+import uk.gov.justice.digital.hmpps.licences.util.TestData
+
+@Stepwise
+class ConsiderationSpec extends GebReportingSpec {
```

Comment:
For consistency should this be `ApprovalConsiderationSpec`?

Reply:
agreed

Reply:
yes